Demo/Application/Base/ContainmentViewController.swift

```diff
@@ -176,10 +176,21 @@ class ContainmentViewController: UIViewController {
                 }
             }
 
+        //TODO Remove this once ModXO goes GA
         case .newPayPalCheckoutTokenizationKey:
-            updateStatus("Fetching new checkout token...")
-            let newPayPalCheckoutTokenizationKey = "sandbox_rz48bqvw_jcyycfw6f9j4nj9c"
-            currentViewController = instantiateViewController(with: newPayPalCheckoutTokenizationKey)
+            updateStatus("Fetching modXO (origami) checkout token...")
+            
+            var tokenizationKey: String = ""
+            switch BraintreeDemoSettings.currentEnvironment {
+            case .sandbox:
+                tokenizationKey = "sandbox_rz48bqvw_jcyycfw6f9j4nj9c"
+            case .production:
+                tokenizationKey = "production_t2wns2y2_dfy45jdj3dxkmz5m"
+            default:
+                tokenizationKey = "development_testing_integration_merchant_id"
```

Comment:
random q: Is `BraintreeDemoSettings.currentEnvironment` nullable?

Reply:
```
enum BraintreeDemoEnvironment: Int {
    case sandbox
    case production
    case custom
}
```

Reply:
We could roll with `case .custom:`. Default is a nice catch-all but it might help at a glance to make the switch case exhaustive.